### PR TITLE
docs: improve readme

### PR DESCRIPTION
**Pull request by edibotopic**
Attempts to improve the readme by providing:

- a title that includes the product name
- a more concise introductory paragraph that explicitly mentions usage on multiple platforms (e.g., desktop, server)
- a section that clearly outlines briefly the content to be found in the documentation wiki, including SSH authentication

When reading, a user should get a more immediate sense of what the product offers and where more technical information can be found, including information on architecture, etc.

UDENG-4080
UDENG-4083

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Welcome to Authentication daemon for external Broker
+# Canonical authd
```
Comment (jibel) [inline README.md:1]: Not sure about this change.
The project's organisation is Ubuntu. Then the title doesn't bring any information just the name of the project and the owner. 

Maybe "authd: an **auth**entication **d**aemon for cloud identity providers"

It is then redundant with the first sentence of the first paragraph which could be removed if we don't want to repeat the statement but it doesn't hurt to keep it.

File: README.md [hunk 2]
```diff
@@ -20,27 +20,39 @@
 [![Go Report Card][goreport-image]][goreport-url]
 [![Reference documentation][reference-documentation-image]][reference-documentation-url]
 
-This is the code repository for authd, an authentication daemon for cloud-based identity provider.
+Authd is an authentication daemon for cloud-based identity providers. It helps
+ensure the secure management of identity and access for Ubuntu machines
+anywhere in the world, on desktop and the server. Authd's modular design makes
+it a versatile authentication service that can integrate with multiple identity
+providers. MS Entra ID is currently supported and support for several other
```
Comment (jibel) [inline README.md:27]: "supported/support" this is a bit redundant. Maybe change with

MS Entra ID is currently supported, and several other identity providers are under active development.

File: README.md [hunk 3]
```diff
@@ -20,27 +20,39 @@
...
+
+Authd uses brokers to interface with cloud identity providers through a [DBus API](https://github.com/ubuntu/authd/blob/HEAD/examplebroker/com.ubuntu.auth.ExampleBroker.xml).
+
+Currently [MS Entra ID](https://learn.microsoft.com/en-us/entra/fundamentals/whatis) is supported as an identity provider. 
 The [MS Entra ID broker](https://github.com/ubuntu/oidc-broker) allows you to authenticate against MS Entra ID using MFA and the device authentication flow.
- 
-## Installation and usage
 
-The [documentation](https://github.com/ubuntu/authd/wiki/01---Get-started-with-authd) provides detailed instructions for installing, configuring, and using Authd and the MS Entra ID provider.
+For development purposes, authd also provides an example broker to help you develop your own.
```
Comment (jibel) [inline README.md:47]: link to https://github.com/ubuntu/authd/tree/main/examplebroker

File: README.md [hunk 4]
```diff
@@ -20,27 +20,39 @@
...
 
 ## Get involved
 
 This is an [open source](COPYING) project and we warmly welcome community contributions, suggestions, and constructive feedback. If you're interested in contributing, please take a look at our [Contribution guidelines](CONTRIBUTING.md) first.
 
-- to report an issue, please file a bug report against our repository, using a bug template.
-- for suggestions and constructive feedback, report a feature request bug report, using the proposed template.
+- To report an issue, please file a bug report against our repository, using the **report an issue** template.
```
Comment (jibel) [inline README.md:53]: link to https://github.com/ubuntu/authd/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml&title=Issue%3A+

File: README.md [hunk 5]
```diff
@@ -20,27 +20,39 @@
...
+- For suggestions and constructive feedback, report a feature request bug report, using the **request a feature** template.
```
Comment (jibel) [inline README.md:54]: link to https://github.com/ubuntu/authd/issues/new?assignees=&labels=feature&projects=&template=feature_request.yml&title=Feature%3A+

Or start with the link to https://github.com/ubuntu/authd/issues/new/choose then the possible options.

**Review by jibel — CHANGES_REQUESTED** [review]
Thanks for these changes. I added  a couple of comments though.

File: README.md (hunk 1 above)
Comment (edibotopic) [reply README.md:1]: I like your suggestion.
The product name should be in the header but the org is not needed.

File: README.md (hunk 2 above)
Comment (edibotopic) [reply README.md:27]: Agreed.

**Comment by edibotopic** [comment]
> Thanks for these changes. I added a couple of comments though.

Much thanks @jibel .
All of your improvements were incorporated in: https://github.com/ubuntu/authd/pull/510/commits/8960da7113502f026339a468bc30337424ee33c8

**Review by jibel — APPROVED** [review]
approved! It's unlikely that the CI failure is due to this PR.

**Pull request merged**